opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -66,7 +67,8 @@ namespace Opm {
             AllProperties = IntProperties | DoubleProperties
         };
 
-        EclipseState(std::shared_ptr< const Deck > deck , const ParseContext& parseContext);
+        EclipseState(std::shared_ptr< const Deck > );
```

Comment:
I am not sure, but I can try.  It would imply that EclipseGrid can be constructed from a reference.  I don't think it is impossible, but I don't know how far that would propagate.  Investigating now.